kinto_wizard/yaml2kinto.py

```diff
@@ -142,6 +144,25 @@
                                                 collection=collection_id,
                                                 data=record_data,
                                                 permissions=record_permissions)
+                if delete_missing_records and collection_records and \
+                   'records' in current_collection:
+                    # Fetch all records IDs
+                    file_records_ids = set(collection_records.keys())
+                    server_records_ids = set(current_collection['records'].keys())
+
+                    to_delete = server_records_ids - file_records_ids
+                    if not force:
+                        message = ("Are you sure that you want to delete the "
+                                   "following {} records?".format(len(list(to_delete))))
+                        value = input(message)
+                        print(to_delete)
+                        if value.lower() not in ['y', 'yes']:
+                            print("Exiting")
+                            exit(1)
+                    for record_id in to_delete:
+                        batch.delete_record(id=record_id,
+                                            bucket=bucket_id,
+                                            collection=collection_id)
```

Comment:
It would be nice if we could generalize it to bucket/collections/groups

Reply:
For groups it might make sense, but for buckets and collections I would rather not trust kinto-wizard for such a dangerous task :joy:

Reply:
I have created https://github.com/Kinto/kinto-wizard/issues/48 for that purpose. I believe we might need specifics option for each ``--delete-records`` ``--delete-groups`` ``--delete-collections`` etc.

Reply:
> I would rather not trust kinto-wizard for such a dangerous task joy

Why would be buckets so different? If we prompt, I don't see the problem...